src/index.html

```diff
@@ -28,31 +28,6 @@
     </footer>
 
     <div id="comment-popup">
```

Comment:
[OPTIONAL] As a suggestion, we can even remove this element from DOM and make it on the fly in js. This way the html markup remains clean. What do you think?

This can be done later.